src/components/MenuPanel/MenuPanel.js

```diff
@@ -65,11 +65,42 @@ function MenuPanel({
   const [systemAppsOpened, setSystemAppsOpened] = useState(
     systemAppsOpenedState.isOpen()
   )
-  const [systemAppsToggled, setSystemAppsToggled] = useState(false)
+  const [showSystemsAppsAnimation, setShowSystemsAppsAnimation] = useState(
```

Comment:
Renamed this as it was previously being used to track whether a user had interacted with the systems app toggle, but now this value is being set automatically based on `isSystemAppActive` to control the animation setting.